troupon/static/js/map.js

```diff
@@ -0,0 +1,41 @@
+<script type="text/javascript">
```

Comment:
Expected an identifier and instead saw '<'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.